details.php

```diff
@@ -66,18 +68,21 @@
 				<tbody>
 				<?php
 					$sl = 1;
-					while ($row = mysqli_fetch_array($result_3, MYSQLI_ASSOC))
+					while ($row = DatabaseConnection::db_fetch_array($result_list))
 					{
 						echo "<tr>";
 						echo "<td>".$sl++."</td>";
 						foreach ($row as $key => $value)
 						{
 							if('comm_id' == $key)
 							{
-								$q_comm = "SELECT GROUP_CONCAT(type SEPARATOR ', ') FROM 
-								`communication_medium` WHERE id IN ($value)";
-								$result_4 = mysqli_query($conn, $q_comm);
-								while ($row1 = mysqli_fetch_array($result_4, MYSQLI_ASSOC))
+								// print_r($obj);exit;
```

Comment:
Please decide what you are using for comparison
1) if('comm_id' == $key) 
2) if($key == 'comm_id')

As in previous files I have seen you following 2. So use it everywhere